python/delta_sharing/reader.py

```diff
@@ -85,11 +89,73 @@ def limit(self, limit: Optional[int]) -> "DeltaSharingReader":
             timestamp=self._timestamp
         )
 
+    def to_pandas_kernel(self):
+        self._rest_client.set_delta_format_header()
```

Comment:
let's correct the name of this function. it is actually setting both format in the header.

Reply:
and if needed, create a function `set_delta_format_header` which only set delta format in the header.

Reply:
done changed it to set_sharing_capabilities_header()